index.js

```diff
@@ -0,0 +1,75 @@
+const { Assyst } = require('./lib/Assyst.js');
+const config = require('./config.json');
+
+const client = new Assyst(config);
+
+console.log('Starting Assyst');
+
+console.log('Loading commands');
+
+client.loadCommands();
+
+async function checkRepl(msg) {
+    if (!msg.content.startsWith('`') || !msg.content.endsWith('`') ) {
+        return;
+    } if (msg.content.startsWith('```') ) {
+        return;
+    } if (!client.repl.map(i => i.user).includes(msg.author.id) || !client.repl.map(i => i.channel).includes(msg.channel.id) ) {
+        return;
+    }
+    const currentRepl = client.repl.find(j => j.user === msg.author.id && j.channel === msg.channel.id);
```

Comment:
I was not understanding why you were making repl an array and not a Map at first but with this line I do.
I don't think there would actually be a better way to handle this actually.
I will think about whether there is a better solution but this seems pretty optimum i'd say. Good job!!